run/events-pubsub/Program.cs

```diff
@@ -16,25 +16,29 @@
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.Hosting;
 
-namespace HelloEvents
+namespace EventsPubSub
 {
     public class Program
     {
+        // [START run_events_pubsub_server]

```

Comment:
One of the issues we get with samples is that users don't know which using directives to add to their code or where "to put" the code snippets we show them. So we have started to shift towards having a sample per file and including the whole code on the sample, including the using statements.
Would it be possible for this whole file to be just one sample that shows how to setup and start the server so that you can show the usings and the file context as well?
Else, would it possible to separate server setup and start in two different files? I wouldn't like this alternative as muchs because it's different to what MS project templates do.
I understand that this depends some on how the doc page is written and that at this point it might not be possible to change.

Reply:
Yes, the doc is not written yet, but other samples in Python and Node.js use these directives..so I'm assuming doc will have specific sections to pull the code in. I don't think I have much freedom in simplifying or removing these tags, I'm just following Grant's proposal on these sample.